apps/publisher/themes/default/js/list_assets.js

```diff
@@ -189,7 +189,9 @@ var createQuery = function(options) {
     var input = $('#inp_searchAsset').val();
     var category = options.category || undefined;
     var searchQueryString = '?';
-    input = setDefaultSearchQuery(input);
+    if (input.indexOf(":") == -1 && input.trim() !== "") {
```

Comment:
Why are we checking for an empty string since this will never evaluate to true if the first case is true.